addOns/sstiscanner/src/main/java/org/zaproxy/zap/extension/sstiscanner/ExtensionSSTiScanner.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2019 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.extension.sstiscanner;
+
+import org.parosproxy.paros.Constant;
+import org.parosproxy.paros.extension.ExtensionAdaptor;
+
+/**
+ * A helper extension to force the load, done by core, of the resource messages bundled in the
+ * add-on.
+ */
```

Comment:
The extension is no longer needed to load the resource messages, they can be declared in the manifest.

Reply:
https://github.com/zaproxy/zap-extensions/blob/85bac82a326ed147772ea2a7b14d7bdab9299780/addOns/retire/retire.gradle.kts#L13-L16

Reply:
I did this, however I also had to:
```java
    @TestFactory
    @Disabled
    Collection<DynamicTest> commonScanRuleTests() {
        // Disabled shouldHaveI18nNonEmptyName doesn't seem to be working with no extension
        return null;
    }
```

Reply:
Even after moving to ascanrulesAlpha this still fails if re-enabled. Thoughts on that @thc202?

Reply:
Sorry for the late follow up, the test classes should be now extending `ActiveScannerTest` instead of `ActiveScannerTestUtils`. (In the previous case it would have to call `mockMessages(String, String)`.)

Reply:
Thanks, I'll touch that up, and see how it goes.